src/node.rs

```diff
@@ -601,6 +601,6 @@ macro_rules! connect {
     };
     ($x:expr => $y:expr => $($rest:tt)+) => {
         $y.set_parent(&$x);
-        connect!($y => $($rest)*);
+        $crate::connect!($y => $($rest)*);
```

Comment:
This isn't strictly necessary for this PR's purpose, but it allows you to use `connect!` without directly importing `connect!`, like: `k::connect!`. Support for such macro calls would be useful when removing the use of `#[macro_use] extern crate`.